fahrschein/src/main/java/org/zalando/fahrschein/domain/SubscriptionRequest.java

```diff
@@ -32,16 +30,18 @@ public String toString() {
     private final Set<String> eventTypes;
     private final String consumerGroup;
     private final Position readFrom;
+    private final List<Cursor> initialCursors;
 
-    public SubscriptionRequest(String owningApplication, Set<String> eventTypes, String consumerGroup, Position readFrom) {
+    public SubscriptionRequest(String owningApplication, Set<String> eventTypes, String consumerGroup, Position readFrom, List<Cursor> initialCursors) {
         this.owningApplication = owningApplication;
         this.eventTypes = unmodifiableSet(eventTypes == null ? emptySet() : new HashSet<>(eventTypes));
         this.consumerGroup = consumerGroup;
         this.readFrom = readFrom;
+        this.initialCursors = unmodifiableList((initialCursors == null) ? emptyList() : initialCursors);
```

Comment:
This only creates an unmodifiable wrapper, the underlying `initialCursors` parameter could still be modified. Better to create a copy and wrap that in an unmodifiable list.